plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/TransactionLogParser.java

```diff
@@ -200,9 +209,12 @@ public static Object deserializeColumnValue(DeltaLakeColumnHandle column, String
                 // date values are represented as yyyy-MM-dd
                 return LocalDate.parse(valueString).toEpochDay();
             }
-            if (type.equals(createTimestampWithTimeZoneType(3))) {
+            if (type.equals(TIMESTAMP_MICROS)) {
                 return timestampReader.apply(valueString);
```

Comment:
The semantics of `timestampReader` parameter changed, which is good from trino type names perspective, but should come as a prep change. otherwise it LOOKS like commit adding "timestamp WITHOUT tz" support adds `timestampWithZoneReader`.